public/app/features/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplatesTable/AlertRuleTemplatesTable.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { useTable, Column } from 'react-table';
+import { Spinner, useStyles } from '@grafana/ui';
+import { getStyles } from './AlertRuleTemplatesTable.styles';
+import { css } from 'emotion';
+import { logger } from '@percona/platform-core';
+import { AlertRuleTemplateService } from '../AlertRuleTemplate.service';
+import { formatTemplates } from './AlertRuleTemplatesTable.utils';
+import { FormattedTemplate } from './AlertRuleTemplatesTable.types';
+import { Messages } from '../../../IntegratedAlerting.messages';
+
+const { noData, columns } = Messages.alertRuleTemplate.table;
+
+const { name: nameColumn, source: sourceColumn, createdAt: createdAtColumn } = columns;
+
+export const AlertRuleTemplatesTable = () => {
+  const style = useStyles(getStyles);
+  const [pendingRequest, setPendingRequest] = useState(false);
+  const [data, setData] = useState<FormattedTemplate[]>([]);
+
+  const getAlertRuleTemplates = async () => {
+    setPendingRequest(true);
+    try {
+      const { templates } = await AlertRuleTemplateService.list();
+      setData(formatTemplates(templates));
+    } catch (e) {
+      logger.error(e);
+    } finally {
+      setPendingRequest(false);
+    }
+  };
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: nameColumn,
+        accessor: 'summary',
+        width: '70%',
+      } as Column,
+      {
+        Header: sourceColumn,
+        accessor: 'source',
+        width: '20%',
+      } as Column,
+      {
+        Header: createdAtColumn,
```

Comment:
I think that `sourceColumn` and `createdAtColumn` will grow way too much in width on large screens, which is not great. Making the widths constant would help address that. The name column should grow/shrink though.

Reply:
Ok, we agreed to find the right way to do it, so resolving this now.